content/videos/challenges/123.1-chaos-game/index.json

```diff
@@ -0,0 +1,76 @@
+{
+    "title": "Chaos Game Part 1",
+    "description": "In this two-part coding challenge, I implement the \u201cChaos Game\u201d in the p5.js web editor. This first video uses three seed points and results in the famous Sierpinski Triangle.",
+    "videoNumber": "123.1",
+    "videoId": "7gNzMtYo9n4",
+    "date": "2018-11-28",
+    "languages": ["p5.js", "javascript"],
+    "topics": [
+        "chaos game",
+        "fractal",
+        "sierpinski triangle",
+        "iterative function system"
+    ],
+    "canContribute": true,
+    "relatedChallenges": [],
+    "timestamps": [],
+    "codeExamples": [
+        {
+            "title": "Chaos Game",
+            "description": "My visualization of the chaos game.",
+            "image": "chaos.jpg",
+            "urls": {
+                "p5": "https://editor.p5js.org/codingtrain/sketches/BJqKF9KRQ",
+                "processing": "https://github.com/CodingTrain/Coding-Challenges/tree/main/123_ChaosGame_1/Processing"
+            }
+        }
+    ],
+    "groupLinks": [
+        {
+            "title": "References",
+            "links": [
+                {
+                    "icon": "📚",
+                    "title": "Algorithm Archive",
+                    "url": "https://www.algorithm-archive.org/",
+                    "description": "Arcane Algorithm Archive"
+                },
+                {
+                    "icon": "🔗",
+                    "title": "Chaos Game on Wikipedia",
+                    "url": "https://en.wikipedia.org/wiki/Chaos_game",
+                    "description": "Discussion of the chao game."
+                },
+                {
+                    "icon": "💿",
+                    "title": "Refactor by Espen Sande Larsen",
+                    "url": "https://soundcloud.com/espen-sande-larsen-365984601/refactor",
+                    "description": "Refactor song."
+                }
+            ]
+        },
+        {
+            "title": "Videos",
+            "links": [
+                {
+                    "icon": "🚂",
+                    "title": "Chaos Game Part 2",
```

Comment:
This should go in related challenges? See #262

Reply:
Yes, I wasn't sure how do deal with this.  I can change it back.